sections/Pools/PurchasePool/PurchasePoolSection/PurchasePoolSection.tsx

```diff
@@ -43,10 +41,9 @@ const PurchasePool: FC<PurchasePoolProps> = ({ pool }) => {
 	const { monitorTransaction } = TransactionNotifier.useContainer();
 	const [gasLimitEstimate, setGasLimitEstimate] = useState<GasLimitEstimate>(null);
 
-	const { gasPrice, setGasPrice, txState, setTxState, txType, setTxType } =
-		TransactionData.useContainer();
+	const { gasPrice, setTxType, setTxState } = TransactionData.useContainer();
 	const [isMaxValue, setIsMaxValue] = useState<boolean>(false);
-	const [inputValue, setInputValue] = useState(0);
+	const [inputValue, setInputValue] = useState('');
```

Comment:
is there a problem with using 0 as the initial state? number | null is always another option if we need a different state for some reason